assets/stylesheets/bootstrap/mixins/_buttons.scss

```diff
@@ -3,10 +3,10 @@
 // Easily pump out default styles, as well as :hover, :focus, :active,
 // and disabled options for all buttons
 
-@mixin button-variant-plain($color, $background, $border) {
```

Comment:
The `$border` seemed to always be the same as `$color` so I've removed and used `currentColor` for all instances of both `$color` and `$border` except for the base rule `color: $color;` which will get overridden by customer branding if needed